### PR TITLE
feat(accordion, expandablesection): rename displaySize prop value

### LINKED ISSUE #138: Accordion, ExpandableSection - update displaySize 'large' to 'lg'
Follow up to breaking change PR https://github.com/patternfly/patternfly-react/pull/8206

Anywhere a consumer has set `displaySize` to `large` in the Accordion or Expandable section, it needs to be updated to `lg`

_Required actions:_
1. Build codemod
2. Build test
3. Update readme with description & example

### PR DESCRIPTION
**Pull request by jenny-s51**
closes #138

### CODE REVIEW

File: packages/eslint-plugin-pf-codemods/test/rules/v5/accordion-rename-displaySize-large.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const ruleTester = require("../../ruletester");
+const rule = require("../../../lib/rules/v5/accordion-rename-displaySize-large");
+
+ruleTester.run("accordion-rename-displaySize-large", rule, {
+  valid: [
+    {
+      code: `import { Accordion } from '@patternfly/react-core'; <Accordion />`,
```
Comment (gitdallas) [inline packages/eslint-plugin-pf-codemods/test/rules/v5/accordion-rename-displaySize-large.js:7]: should test dislpaySize="lg" for new valid

File: packages/eslint-plugin-pf-codemods/test/rules/v5/accordion-rename-displaySize-large.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    {
+      // No @patternfly/react-core import
+      code: `<Accordion displaySize />`,
```
Comment (gitdallas) [inline packages/eslint-plugin-pf-codemods/test/rules/v5/accordion-rename-displaySize-large.js:11]: should be displaySize="large" to test it won't affect nonpf

File: packages/eslint-plugin-pf-codemods/lib/rules/v5/expandable-section-rename-displaySize-large.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const { renameProp } = require('../../helpers');
+
+// https://github.com/patternfly/patternfly-react/pull/8206
+module.exports = {
+  meta: { fixable: 'code' },
+  create: renameProp(
```
Comment (gitdallas) [inline packages/eslint-plugin-pf-codemods/lib/rules/v5/expandable-section-rename-displaySize-large.js:6]: this is for renaming the property, not changing the property value

**Review by gitdallas — CHANGES_REQUESTED** [review]

**Review by gitdallas — COMMENTED** [review]
Wonder if it we should create a renamePropValue helper function?

**Review by gitdallas — APPROVED** [review]

**Review by thatblindgeye — COMMENTED** [review]
Looks good! Just merge conflicts to resolve, and placing contents in the README alphabetically (could be worth doing in the index file as well)

**Pull request merged**